config/jobs/kubernetes/sig-testing/make-test.yaml

```diff
@@ -173,4 +173,4 @@ periodics:
             - bash
             - -c
           args:
-            - 'make test KUBE_RACE=-race KUBE_TIMEOUT=--timeout=300s GOFLAGS=-count=10'
+            - 'make test KUBE_RACE=-race KUBE_TIMEOUT=--timeout=3000s GOFLAGS=-count=10'
```

Comment:
xref: https://github.com/golang/go/issues/17337

Reply:
This is 50 minutes per test though .... I'm not sure this is reasonable.
We should probably not let the tests take so long? 🤔

Reply:
let's put 600s, so we can start to cut at the 1 min per execution now, and in the next iteration we limit all tests to 30 seconds max, does it make sense?